### PR TITLE
Docs - Updated Submit your exercise section

### LINKED ISSUE #56: Incorrect account name in README
Readme of this repository guides in the [**How to submit your exercise**](https://github.com/form3tech-oss/interview-accountapi#how-to-submit-your-exercise) section you should invite the `@form3tech-interviewer-1` when you finish the exercise.

However, when I copy-pasted this account name to the GitHub form, I got a message this account doesn't exist.

Screenshot 2021-10-03 at 08 34 38

Removing `@` prefix makes it working

Screenshot 2021-10-03 at 08 34 50

I suggest changing this sentence as follows:

- [Invite](https://help.github.com/en/articles/inviting-collaborators-to-a-personal-repository) **form3tech-interviewer-1** to your private repo

### PR DESCRIPTION
**Pull request by adithyaakrishna**
Signed-off-by: Adithya Krishna <aadithya794@gmail.com>

Description:
This PR Fixes #56

### CODE REVIEW

**Comment by adithyaakrishna** [comment]
@ewilde @liamg Could you please review this PR? Thanks in advance :)

**Comment by liamg** [comment]
@adithyaakrishna I'm afraid I no longer work for FORM3 so can't merge this.

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The finished solution **should not:**
 
 - Include your name in the README. If you are new to Go, please also mention this in the README so that we can consider this when reviewing your exercise
 - Create a private [GitHub](https://help.github.com/en/articles/create-a-repo) repository, by copying all files you deem necessary for your submission
-- [Invite](https://help.github.com/en/articles/inviting-collaborators-to-a-personal-repository) @form3tech-interviewer-1 to your private repo
+- [Invite](https://help.github.com/en/articles/inviting-collaborators-to-a-personal-repository) **_form3tech-interviewer-1_** to your private repo
```
Comment (ewilde-form3) [inline README.md:47]:
```suggestion
- [Invite](https://help.github.com/en/articles/inviting-collaborators-to-a-personal-repository) [@form3tech-interviewer-1](https://github.com/form3tech-interviewer-1) to your private repo
```

**Review by ewilde-form3 — CHANGES_REQUESTED** [review]
@adithyaakrishna thank you for your PR. I've made a small suggestion which links to the users profile. The @ syntax is in the correct syntax, hopefully this makes it clearer

**Review by ewilde-form3 — APPROVED** [review]

**Pull request merged**